REFERENCE.md

```diff
@@ -205,12 +205,18 @@ The cross-validation procedure works as follows:
 
 **Note:** this feature is currently marked 'experimental'. Its implementation may change in future releases.
 
-Ghosal and Hooker (2018), show how a boosting step can reduce bias in random forest predictions. We have included a  `boosted_regression_forest` feature, which trains a series of forests, each of which are trained on the OOB residuals from the previous step. `boosted_regression_forest` includes the same parameters as `regression_forest`, which are passed directly to the forest trained in each step. It also includes control over the step selection procedure:
+Ghosal and Hooker (2018), show how a boosting step can reduce bias in random forest predictions. We have included a  `boosted_regression_forest` feature, which trains a series of forests, each of which are trained on the OOB residuals from the previous step. `boosted_regression_forest` includes the same parameters as `regression_forest`, which are passed directly to the forest trained in each step.
 
-- If `boost.steps` is not specified, then a cross-validation procedure selects the number of boosting steps automatically, where the next step is only taken if the estimated OOB error for the next step, computed using a small forest of size `boost.trees.tune`, has an estimated OOB error reduction from  the previous step of more than `boost.error.reduction`. For computational considerations only up to `boost.max.steps` are considered.
-- `boost.steps` can also be specified by the user.
+For computational considerations, if `tune.parameters=TRUE`, then parameters are selected by the `regression_forest` cross-validation only once, in the first step. The selected parameters are then applied to forests in any further steps, without repeating cross-validation. `boosted_regression_forest` also includes control over the step selection procedure. By default, the number of boosting steps is automatically chosen through a cross-validation procedure. This procedure works as follows:
```

Comment:
One last comment: could we move discussion of `tune.parameters` after the description of the boosting procedure? Tuning is not actually enabled by default and I think it is a detail that's best left to the end. Also do you mean "selected only using the first boosting step, when a complete regression forest is trained" as opposed to "selected by the `regression_forest` cross-validation only once"? Because this fact applies to boosting no matter whether cross-validation or a fixed number of boosting steps is used.

Reply:
I meant the tuning procedure in regression_forest, not the cross validation for boosting. Changed the wording and hopefully is more clear now.